modules/programs/broot.nix

```diff
@@ -44,233 +180,50 @@ in {
-        }
-        {
-          invocation = "edit";
-          shortcut = "e";
-          execution = "$EDITOR {file}";
-        }
-        {
-          invocation = "create {subpath}";
-          execution = "$EDITOR {directory}/{subpath}";
-        }
-        {
-          invocation = "view";
-          execution = "less {file}";
-        }
-      ];
-      example = literalExpression ''
-        [
-          { invocation = "p"; execution = ":parent"; }
-          { invocation = "edit"; shortcut = "e"; execution = "$EDITOR {file}" ; }
-          { invocation = "create {subpath}"; execution = "$EDITOR {directory}/{subpath}"; }
-          { invocation = "view"; execution = "less {file}"; }
-          {
-            invocation = "blop {name}\\.{type}";
-            execution = "/bin/mkdir {parent}/{type} && /usr/bin/nvim {parent}/{type}/{name}.{type}";
-            from_shell = true;
-          }
-        ]
-      '';
-      description = ''
-        Define new verbs. For more information, see
-        <link xlink:href="https://dystroy.org/broot/documentation/configuration/#verb-definition-attributes"/>.
-        </para><para>
-        The possible attributes are:
-        </para>
-
-        <para>
-        <variablelist>
-          <varlistentry>
-            <term><literal>invocation</literal> (optional)</term>
-            <listitem><para>how the verb is called by the user, with placeholders for arguments</para></listitem>
-          </varlistentry>
-          <varlistentry>
-            <term><literal>execution</literal> (mandatory)</term>
-            <listitem><para>how the verb is executed</para></listitem>
-          </varlistentry>
-          <varlistentry>
-            <term><literal>key</literal> (optional)</term>
-            <listitem><para>a keyboard key triggering execution</para></listitem>
-          </varlistentry>
-          <varlistentry>
-            <term><literal>shortcut</literal> (optional)</term>
-            <listitem><para>an alternate way to call the verb (without
-            the arguments part)</para></listitem>
-          </varlistentry>
-          <varlistentry>
-            <term><literal>leave_broot</literal> (optional)</term>
-            <listitem><para>whether to quit broot on execution
-            (default: <literal>true</literal>)</para></listitem>
-          </varlistentry>
-          <varlistentry>
-            <term><literal>from_shell</literal> (optional)</term>
-            <listitem><para>whether the verb must be executed from the
-            parent shell (default:
-            <literal>false</literal>)</para></listitem>
-          </varlistentry>
-        </variablelist>
-      '';
-    };
-
     package = mkOption {
       type = types.package;
       default = pkgs.broot;
       defaultText = literalExpression "pkgs.broot";
       description = "Package providing broot";
     };
 
-    skin = mkOption {
-      type = types.attrsOf types.str;
+    config = mkOption {
```

Comment:
We generally prefer using a `settings` option for structured settings instead of `config`.